src/src/com/task33/Task33.java

```diff
@@ -0,0 +1,39 @@
+package src.com.task33;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task33 {
+    public static void main(String[] args) {
+        try {
```

Comment:
You forgot to close stream. You can create stream inside try resources block - in this case it will be closed automatically